recipe/bld.bat

```diff
@@ -1,9 +1,18 @@
-cmake -G "NMake Makefiles" -D EXPECTED_ENABLE_TESTS=OFF -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% %SRC_DIR%
+:: cmd
 
+:: Isolate the build.
+mkdir Build
+cd Build
 if errorlevel 1 exit 1
 
-nmake
-if errorlevel 1 exit 1
+:: Generate the build files.
+cmake .. -G"Ninja" %CMAKE_ARGS% ^
+    -DEXPECTED_ENABLE_TESTS=OFF ^
```

Comment:
I think we should enable the author's tests and run them (with ctest?). This assumes they are validation tests.

Reply:
Added the tests. (Catch2)